README.md

```diff
@@ -243,22 +256,50 @@ IPU ES2K target.
    ...
    ```
 
-5. Pull images for use by Kubernetes Container Runtime Interface (CRI):
+6. Pull images for use by Kubernetes Container Runtime Interface (CRI):
    ```bash
    crictl pull localhost:5000/inframanager:latest
    crictl pull localhost:5000/infraagent:latest
    ```
 
-6. Generate the certificates required for the mTLS connection between infraagent,
+7. Generate the certificates required for the mTLS connection between infraagent,
    inframanager, and infrap4d:
    ```bash
    make gen-certs
    ```
    Note that the above script generates the default keys and certificates and
-   uses cipher suites as specified in the `inframanager/config.yaml` file. If you
-   do not wish to use these default keys, certificates, and cipher suites, then
-   modify the `scripts/mev/tls/gen_certs.sh` script accordingly and modify the
-   `inframanager/config.yaml` file with preferred cipher suites.
+   uses cipher suites as specified in the `inframanager/config.yaml` file.
+   Refer to section [inframanager/config.yaml file update](#inframanagerconfigyaml-file-update)
+   for any custom cipher suite, key, certificate change.
+
+   Note that the above script generates the default keys and certificates and
+   uses cipher suites as specified in the `inframanager/config.yaml` file.
+
+#### inframanager/config.yaml file update
+
+The config.yaml file is used to define the parameters which the inframanager will
+use for the connection establishment with infrap4d and for the interfaces created.
+
+
+InfraManager section:
+arp-mac: The arp-mac needs to be configured. This should be the
+MAC of the interface the user wants to configure as the ARP-proxy gateway.
+This is the address of the interface which is given to the arp-proxy
+namespace using the `scrips/arp_proxy.sh` script
+
+conn
+server-cert
+server-key
+ca-cert
+ciphersuites:
+ Information about the above are detailed in the comments in the config.yaml file
+
+ If you do not wish to use these default keys, certificates, and cipher suites, then
+ modify the `scripts/mev/tls/gen_certs.sh` script accordingly before running
+ `make gen-certs` and modify the `inframanager/config.yaml` file with preferred
+ cipher suites. This change needs to be done in step to update config.yaml file
```

Comment:
I think, it's not coming out clean and below should be removed.

"This change needs to be done in step to update config.yaml file" 
Rather just mention, these changes need to be done prior to creation of docker images.

Reply:
please check